### ISSUE TITLE
Donations

### ISSUE THREAD

**Issue by realappie** [quicktype/quicktype#1608]
I personally really love this project, and think another 5700 developers do if you look at the number of stars 👍 I'm wondering if there is anyway we could show our support, as I would hate to see this project die to lack of time/funds😊 Perhaps its worth looking into github sponsors [here](https://github.com/sponsors#countries)

**Comment by Ziao** [comment]
Agreed, wouldn't mind donating a few beers for the time this saves me :)

**Comment by dvdsgl** [comment]
Thank you!
